lighthouse-core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -14,22 +14,23 @@ const FullPageScreenshotGatherer = require('../../../gather/gatherers/full-page-
  */
 function createMockDriver({contentSize, screenSize, screenshotData}) {
   return {
-    evaluateAsync: async function(code) {
-      if (code === 'window.innerWidth') {
+    evaluate: async function(fn) {
+      if (fn.name === 'resolveNodes') {
         return contentSize.width;
```

Comment:
Not sure I understand this. Shouldn't this be returning a rect?

Reply:
Yeeah this doesn't make sense anymore. never actually did I suppose https://github.com/GoogleChrome/lighthouse/commit/3efbf94c96651a936ce64d81314c100c58ad86a1 no usage of "innerWidth" anywhere. oops.